templates/typescript_gapic/_util.njk

```diff
@@ -16,6 +16,8 @@ limitations under the License.
 
 -#}
 
+{% set regExp = r/\[([^\]]+)\]\[([^\]]+)\]/ %}
```

Comment:
let's give it some meaningful name, e.g. `markdownReferenceRegex`

Reply:
it can be written in a more readable way by using _minimal matching quantifiers_: `.*?` matches any character, but minimum possible amount to make a match. So `\[   (  [^\]+  )  \]` (spaces added for readability) is equal to `\[  (  .*?   )  \]`.

Reply:
agree